pkgs/tools/archivers/unar/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv, fetchurl, gnustep, gnustep-make, unzip, clang, bzip2 }:
+
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
+  pname = "unar";
+  version = "1.10.1";
+
+  src = fetchurl {
+    url = "http://unarchiver.c3.cx/downloads/${pname}${version}_src.zip";
+    sha256 = "0aq9zlar5vzr5qxphws8dm7ax60bsfsw77f4ciwa5dq5lla715j0";
+  };
+
+  buildInputs = [
+    gnustep-make unzip clang gnustep.base bzip2.dev
```

Comment:
You should probably replace gnustep-make with gnnustep.make just so we can have things consistent.